src/list.tsx

```diff
@@ -386,22 +367,33 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
         .filter(Boolean)
         .join(' ');
 
+    const ariaLabel = ariaLabelProp ?? (props.href || props.to) ? computedAriaLabelForLink : undefined;
+
     const radioContext = useRadioContext();
     const disabled = props.disabled || (props.radioValue !== undefined && radioContext.disabled);
     const hasHoverDefault = !disabled && !isInverse;
     const hasHoverInverse = !disabled && isInverse;
+    const hasControl = hasControlProps(props);
+    const isInteractive = !!props.onPress || !!props.href || !!props.to;
+    const hasChevron = hasControl ? false : withChevron ?? isInteractive;
+
+    const interactiveProps = {
+        href: props.href,
+        newTab: props.newTab,
+        loadOnTop: props.loadOnTop,
+
+        to: props.to,
+        fullPageOnWebView: props.fullPageOnWebView,
+        replace: props.replace,
+
+        onNavigate: props.onNavigate,
+        onPress: props.onPress,
+        trackingEvent: props.trackingEvent,
+    } as unknown as TouchableProps;
```

Comment:
If there is a cleaner way to do this, I'm all ears :)

Reply:
the alternative way is as it is in master, not merging the to/href/onPress cases. Another option that I think could work is by destructuring all props but interactiveProps, something like this: https://github.com/Telefonica/mistica-web/blob/master/src/text-link.tsx#L40

Reply:
Discussed offline. I've removed `as unknown` to avoid weakening TS type safety.